policy/modules/system/systemd.fc

```diff
@@ -50,6 +50,7 @@ HOME_DIR/\.config/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_unit
 /usr/lib/systemd/system/.*sleep.*\.(service|target)		--	gen_context(system_u:object_r:power_unit_file_t,s0)
 /usr/lib/systemd/system/.*shutdown.*\.(service|target)		--	gen_context(system_u:object_r:power_unit_file_t,s0)
 /usr/lib/systemd/system/.*suspend.*\.(service|target)		--	gen_context(system_u:object_r:power_unit_file_t,s0)
+/usr/lib/systemd/system-sleep(/.*)		gen_context(system_u:object_r:systemd_sleep_exec_t,s0)
```

Comment:
The `()`s are not needed. And there should be a `--` so that directories are not matched.

Also, I'd move it one line below so that it's not in the middle of unit file fcontexts.